### PR TITLE
Report coverage unconditionally on Travis

### PR DESCRIPTION
**Pull request by greg0ire**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | n/a

#### Summary

pcov is fast enough for this to be acceptable.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,10 +8,7 @@ cache:
 
 before_install:
   - phpenv config-rm xdebug.ini || true
-  - |
-    if [ "x$COVERAGE" == "xyes" ]; then
-      pecl install pcov-1.0.0
-    fi
+  - pecl install pcov-1.0.0
```
Comment (morozov) [inline .travis.yml:11]: Can we do just `pecl install pcov` and use the latest stable? It may be required for compatibility with the latest PHP versions.

**Comment by greg0ire** [comment]
This raises the question about the Smoke Testing step… should I keep it and remove the sqlite job from test? That way if something is wrong with the unit tests we don't run a bazillion tests.

**Comment by morozov** [comment]
> This raises the question about the Smoke Testing step… should I keep it and remove the sqlite job from test? That way if something is wrong with the unit tests we don't run a bazillion test jobs for nothing

Definitely. That's the idea. If there's a unit test failure, you don't want to wait 10 minutes until the DB2 job completes. Ideally, I'd like to not run tests until all static analysis phases succeed (which was the case before we moved some jobs to Actions) but I'm not sure how this could be implemented.

**Comment by greg0ire** [comment]
I think this will be achieved by moving to GA, with [job dependencies](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs). BTW @bendavies , are you still planning to work on that? :pray: :crossed_fingers:

**Comment by morozov** [comment]
Looks good :+1: Please squash.

**Comment by greg0ire** [comment]
> Looks good +1 Please squash.

Should I really? I mean I deliberately didn't, since it does not have to do with running coverage unconditionally… using the latest stable could be something we would want to do or revert independently, or maybe there is a link I don't see?

**Comment by morozov** [comment]
Without the second commit, the first one fails on PHP 7.4.

**Comment by greg0ire** [comment]
Oh all right then, didn't notice.

**Review by morozov — APPROVED** [review]

**Comment by greg0ire** [comment]
Ok now continousphp is flaky too…

**Comment by bendavies** [comment]
> I think this will be achieved by moving to GA, with [job dependencies](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs). BTW @bendavies , are you still planning to work on that?

I do have a branch that moves postgres and myswl tests to GitHub actions.
It needs work.
I can submit it I'd you'd like to work on it together.

**Comment by morozov** [comment]
While I like the idea as such, before we go full steam ahead, could we establish some expectations of the end result? E.g. Travis currently allows us to run 10 jobs in parallel (I believe it's a favor since the [default for OSS is 5](https://twitter.com/travisci/status/651856122559774722), may be outdated) and we execute ~40-50 jobs per build.

Will we be able to do the same on GitHub?

**Comment by bendavies** [comment]
> While I like the idea as such, before we go full steam ahead, could we establish some expectations of the end result? E.g. Travis currently allows us to run 10 jobs in parallel (I believe it's a favor since the [default for OSS is 5](https://twitter.com/travisci/status/651856122559774722), may be outdated) and we execute ~40-50 jobs per build.
> 
> Will we be able to do the same on GitHub?

I think 20 as per https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#usage-limits

There might be other reasons that would prevent migrating.
Last time I checked, secrets were not shared with forks. Are any secrets used?

**Comment by morozov** [comment]
> Last time I checked, secrets were not shared with forks. Are any secrets used?

I believe this is by design on any public CI. No secrets are currently used.

**Pull request merged**